docs/source/developers/guide/resources.rst

```diff
@@ -27,3 +27,52 @@
 ************************************
 Additional information and resources
 ************************************
+
+On this page we will list resources that may be relevant or useful for 
+contributors who want to learn more about different parts of Apache Arrow.
+
+.. Annotation Glossary
+.. -------------------
+.. ARROW-15130
+
+Additional information
+----------------------
+
+- GitHub Actions
+
+  GitHub Actions is a continuous integration and continuous delivery (CI/CD) platform.
+  In Apache Arrow we use GitHub actions to build and test every pull request that is
+  opened, merged etc. To do so there are workflows created and then run via GitHub actions.
+
+  - `Apache Arrow Actions on GitHub <https://github.com/apache/arrow/actions>`_
+  - `Location of the workflows in Arrow: arrow/.github/workflows/ <https://github.com/apache/arrow/tree/master/.github/workflows>`_
+  - `GitHub Documentation on GitHub Actions <https://docs.github.com/en/actions>`_
+  .. ARROW-13841: [Doc] Document the different subcomponents that make up the CI and how they fit together:
+  .. https://github.com/apache/arrow/pull/11821
+
+- Nightly builds
+
+  R package has an option of installing the nightly builds which are daily development
+  builds of the R package and are not the official releases. See more on the
+  `Install R package article <https://ursalabs.org/arrow-r-nightly/articles/install.html#install-the-nightly-build>`_.
+
+- `Apache Arrow releases <https://arrow.apache.org/release/>`_
+
+Other resources
+---------------
+
+- :ref:`contributing`
+- `Arrow R Developer Guide <https://arrow.apache.org/docs/r/articles/developing.html>`_
+- `Writing Bindings article for R package <https://arrow.apache.org/docs/r/articles/developers/bindings.html>`_.
+
+- `Tidyverse: Make a reprex <https://www.tidyverse.org/help/#reprex>`_
+- `Craft Minimal Bug Reports by Matthew Rocklin <https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports>`_
+.. - Blog post walking through writing an R binding (not yet written)
```

Comment:
Is there are JIRA opened for this? If so, then reference it, or maybe add a `TODO` tag.

Reply:
Oh, my mistake, this is a leftover of a merge conflict thing. The article is written and the link is added to this PR in line 66. Will remove this. Thanks for catching it!